runtime/android/core_internal/src/org/xwalk/core/internal/extension/api/contacts/ContactConstants.java

```diff
@@ -38,7 +38,7 @@
     // e.g. When to find "givenName=John", we actually need to find "John" in "givenNames".
     public static final Map<String, String> findFieldMap = createStringMap(new String[]{
         "familyName", "familyNames",
-        "givenName", "givenNames",
+        "givenNames", "givenNames",
```

Comment:
I have to admit that I'm confused by the whole API implementation, so help here is appreciated. http://www.w3.org/TR/contacts-api/ just says the spec has been discontinued and people are not supposed to implement it. Then there is http://www.w3.org/TR/2011/WD-contacts-api-20110616/, which is an older version of the spec, but the `find()` method there does not resemble what our implementation looks like (and all the fields are mentioned in the singular form, like "givenName" instead of "givenNames"). Can you please point to which spec is actually being implemented here?